# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_cache-control_request.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_cache-control_request.js
--- a/netwerk/test/unit/test_cache-control_request.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_cache-control_request.js	Tue Feb 12 16:08:25 2019 +0000
@@ -105,7 +105,7 @@
 add_test(() => {
   // Must not create a cache entry
   var ch = make_channel(resource_age_100_url, "no-store");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(!cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -116,7 +116,7 @@
 add_test(() => {
   // Prepare state only, cache the entry
   var ch = make_channel(resource_age_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -127,7 +127,7 @@
 add_test(() => {
   // Check the prepared cache entry is used when no special directives are added
   var ch = make_channel(resource_age_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -139,7 +139,7 @@
   // Try again, while we already keep a cache entry,
   // the channel must not use it, entry should stay in the cache
   var ch = make_channel(resource_age_100_url, "no-store");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -153,7 +153,7 @@
 add_test(() => {
   // Check the prepared cache entry is used when no special directives are added
   var ch = make_channel(resource_age_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -164,7 +164,7 @@
 add_test(() => {
   // The existing entry should be revalidated (we expect a server hit)
   var ch = make_channel(resource_age_100_url, "no-cache");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -178,7 +178,7 @@
 add_test(() => {
   // Check the prepared cache entry is used when no special directives are added
   var ch = make_channel(resource_age_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -190,7 +190,7 @@
   // The existing entry's age is greater than the maximum requested,
   // should hit server
   var ch = make_channel(resource_age_100_url, "max-age=10");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -202,7 +202,7 @@
   // The existing entry's age is greater than the maximum requested,
   // but the max-stale directive says to use it when it's fresh enough
   var ch = make_channel(resource_age_100_url, "max-age=10, max-stale=99999");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -214,7 +214,7 @@
   // The existing entry's age is lesser than the maximum requested,
   // should go from cache
   var ch = make_channel(resource_age_100_url, "max-age=1000");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_age_100_url), ""));
 
@@ -228,7 +228,7 @@
 add_test(() => {
   // Preprate the entry first
   var ch = make_channel(resource_stale_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_stale_100_url), ""));
 
@@ -241,7 +241,7 @@
   // Check it's not reused (as it's stale) when no special directives
   // are provided
   var ch = make_channel(resource_stale_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_stale_100_url), ""));
 
@@ -252,7 +252,7 @@
 add_test(() => {
   // Accept cached responses of any stale time
   var ch = make_channel(resource_stale_100_url, "max-stale");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_stale_100_url), ""));
 
@@ -263,7 +263,7 @@
 add_test(() => {
   // The entry is stale only by 100 seconds, accept it
   var ch = make_channel(resource_stale_100_url, "max-stale=1000");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_stale_100_url), ""));
 
@@ -275,7 +275,7 @@
   // The entry is stale by 100 seconds but we only accept a 10 seconds stale
   // entry, go from server
   var ch = make_channel(resource_stale_100_url, "max-stale=10");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_stale_100_url), ""));
 
@@ -289,7 +289,7 @@
 add_test(() => {
   // Preprate the entry first
   var ch = make_channel(resource_fresh_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 
@@ -300,7 +300,7 @@
 add_test(() => {
   // Check it's reused when no special directives are provided
   var ch = make_channel(resource_fresh_100_url);
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 
@@ -311,7 +311,7 @@
 add_test(() => {
   // Entry fresh enough to be served from the cache
   var ch = make_channel(resource_fresh_100_url, "min-fresh=10");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(!hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 
@@ -322,7 +322,7 @@
 add_test(() => {
   // The entry is not fresh enough
   var ch = make_channel(resource_fresh_100_url, "min-fresh=1000");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 
@@ -336,7 +336,7 @@
 
 add_test(() => {
   var ch = make_channel(resource_fresh_100_url, "unknown1,unknown2 = \"a,b\",  min-fresh = 1000 ");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 
@@ -346,7 +346,7 @@
 
 add_test(() => {
   var ch = make_channel(resource_fresh_100_url, "no-cache = , min-fresh = 10");
-  ch.asyncOpen2(new ChannelListener(function(request, data) {
+  ch.asyncOpen(new ChannelListener(function(request, data) {
     Assert.ok(hit_server);
     Assert.ok(cache.exists(make_uri(resource_fresh_100_url), ""));
 